ForwardingDecision.cs

```diff
@@ -35,14 +35,7 @@ public abstract class ForwardingDecision {
     /// </summary>
     public sealed class Drop : ForwardingDecision {
       // We only need to keep a single instance of Drop, so we use a singleton pattern.
-      public static readonly Drop inst;
-      private Drop () {}
```

Comment:
You're right. It's now fixed in https://github.com/niksu/pax/pull/7/commits/73f81bd3a82bae5730c4f5595a2311995b5b6370
Good catch!